### ISSUE TITLE
trainning error when using segmenter

### ISSUE THREAD

**Issue by mengxia1994** [open-mmlab/mmsegmentation#2018]
I meet the simular problem like  #1351 (maybe...). I have two classes and i set the num_classes as 3. with pixel label from 0 to 2. It worked well on segformer(ended with a really good performence). But when I try to use segmenter to train, the training looks strange. The decode.acc and loss seems normal, but the val result is nearly always zero.

**Comment by mengxia1994** [comment]
2022-09-02 09:06:45,228 - mmseg - INFO - Iter [650/100000]	lr: 9.947e-04, eta: 1:44:39, time: 0.063, data_time: 0.002, memory: 1485, decode.loss_ce: 0.0028, decode.acc_seg: 93.3247, loss: 0.0028
2022-09-02 09:06:48,408 - mmseg - INFO - Iter [700/100000]	lr: 9.943e-04, eta: 1:44:38, time: 0.064, data_time: 0.002, memory: 1485, decode.loss_ce: 0.0022, decode.acc_seg: 95.3112, loss: 0.0022
2022-09-02 09:06:51,564 - mmseg - INFO - Iter [750/100000]	lr: 9.939e-04, eta: 1:44:35, time: 0.063, data_time: 0.002, memory: 1485, decode.loss_ce: 0.0025, decode.acc_seg: 95.2285, loss: 0.0025
2022-09-02 09:06:54,718 - mmseg - INFO - Iter [800/100000]	lr: 9.935e-04, eta: 1:44:30, time: 0.063, data_time: 0.002, memory: 1485, decode.loss_ce: 0.0035, decode.acc_seg: 94.6487, loss: 0.0035
2022-09-02 09:06:57,876 - mmseg - INFO - Iter [850/100000]	lr: 9.931e-04, eta: 1:44:27, time: 0.063, data_time: 0.002, memory: 1485, decode.loss_ce: 0.0027, decode.acc_seg: 95.4519, loss: 0.0027
2022-09-02 09:07:01,026 - mmseg - INFO - Iter [900/100000]	lr: 9.927e-04, eta: 1:44:23, time: 0.063, data_time: 0.002, memory: 1485, decode.loss_ce: 0.0022, decode.acc_seg: 95.7974, loss: 0.0022
2022-09-02 09:07:04,179 - mmseg - INFO - Iter [950/100000]	lr: 9.923e-04, eta: 1:44:19, time: 0.063, data_time: 0.002, memory: 1485, decode.loss_ce: 0.0023, decode.acc_seg: 96.1351, loss: 0.0023
2022-09-02 09:07:07,319 - mmseg - INFO - Saving checkpoint at 1000 iterations
2022-09-02 09:07:07,622 - mmseg - INFO - Exp name: segmenter_vit-s_linear_1x2_480x360_100k_ade20k.py
2022-09-02 09:07:07,623 - mmseg - INFO - Iter [1000/100000]	lr: 9.919e-04, eta: 1:44:45, time: 0.069, data_time: 0.002, memory: 1485, decode.loss_ce: 0.0023, decode.acc_seg: 96.1714, loss: 0.0023
2022-09-02 09:09:20,277 - mmseg - INFO - per class results:
2022-09-02 09:09:20,278 - mmseg - INFO - 
+------------+-------+-------+
|   Class    |  IoU  |  Acc  |
+------------+-------+-------+
| background | 63.75 | 65.17 |
|    lane    |  0.1  |  1.74 |
|    hole    |  0.0  |  0.0  |
+------------+-------+-------+
2022-09-02 09:09:20,278 - mmseg - INFO - Summary:
2022-09-02 09:09:20,278 - mmseg - INFO - 
+-------+-------+------+
|  aAcc |  mIoU | mAcc |
+-------+-------+------+
| 63.34 | 21.28 | 22.3 |
+-------+-------+------+

**Comment by mengxia1994** [comment]
part of the output is above. After several thousands of iter it is still the same.

**Comment by xiexinch** [comment]
Hi @mengxia1994,
Did you check if the pre-trained model was loaded correctly? I don't have much idea about this problem at the moment, I'll try other datasets with this model and feedback to you ASAP.

**Comment by mengxia1994** [comment]
您好，我已收到您的邮件，我将尽快查阅并给予回复，谢谢！（孟夏的自动回复）

**Comment by mengxia1994** [comment]
@xiexinch Thank you for your reply~ I will do some check. 
I have tried both using the pretrained model and not using it, the only difference is the decode.acc_seg and loss are better and improve faster in the first 1000 iter. When come to the val part, same problem happened.

**Comment by xiexinch** [comment]
Closing the issue, as there is no activity for a while.
We hope your issue has been resolved. 
If not, please feel free to open a new one.
